src/pages/Homepage.tsx

```diff
@@ -1,7 +1,10 @@
+import BarChart from "../components/BarChart";
```

Comment:
it should be added to the `Overview` page instead of the `Home`, no ?

Reply:
Yes! Indeed you're right, I've made the modification.